airbyte-bootloader/src/main/java/io/airbyte/bootloader/SecretMigrator.java

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.bootloader;
+
+import static io.airbyte.config.persistence.split_secrets.SecretsHelpers.COORDINATE_FIELD;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.google.common.annotations.VisibleForTesting;
+import io.airbyte.commons.json.JsonPaths;
+import io.airbyte.commons.json.Jsons;
+import io.airbyte.config.ConfigSchema;
+import io.airbyte.config.DestinationConnection;
+import io.airbyte.config.SourceConnection;
+import io.airbyte.config.StandardDestinationDefinition;
+import io.airbyte.config.StandardSourceDefinition;
+import io.airbyte.config.persistence.ConfigPersistence;
+import io.airbyte.config.persistence.split_secrets.SecretCoordinate;
+import io.airbyte.config.persistence.split_secrets.SecretPersistence;
+import io.airbyte.config.persistence.split_secrets.SecretsHelpers;
+import io.airbyte.validation.json.JsonValidationException;
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.UUID;
+import java.util.concurrent.atomic.AtomicReference;
+import java.util.function.Supplier;
+import java.util.stream.Collectors;
+import lombok.AllArgsConstructor;
+import lombok.Value;
+import lombok.extern.slf4j.Slf4j;
+
+@AllArgsConstructor
+@Slf4j
+public class SecretMigrator {
+
+  private final ConfigPersistence configPersistence;
+  private final SecretPersistence secretPersistence;
+
+  @Value
+  static class ConnectorConfiguration {
+
+    private final UUID workspace;
+    private final JsonNode configuration;
+    private final JsonNode specs;
+
+  }
+
+  public void migrateSecrets() throws JsonValidationException, IOException {
+    final List<StandardSourceDefinition> standardSourceDefinitions = configPersistence.listConfigs(ConfigSchema.STANDARD_SOURCE_DEFINITION,
+        StandardSourceDefinition.class);
```

Comment:
```suggestion
    final List<StandardSourceDefinition> standardSourceDefinitions = configPersistence
        .listConfigs(ConfigSchema.STANDARD_SOURCE_DEFINITION,StandardSourceDefinition.class);
```
args on same line or 1 arg per line.

Reply:
done